### DISCUSSION TITLE
Rename form button

### DISCUSSION THREAD

**Discussion by divdax** [filamentphp/filament#5785] [Help] [answered]
It's possible to rename a form button? (Submit)

**Comment by Sicklou** [comment, marked as answer]
I guess saying hello is fine when asking for help :-)

I think you can overwrite the `getSaveFormAction()` method in your Edit{Model} class.
Without more context I guess it is in this type of use case.

```php

use Filament\Pages\Actions\Action;

protected function getSaveFormAction(): Action
{
    return Action::make('save')
        ->label("MyLabel")
        ->submit('save')
        ->keyBindings(['mod+s']);
}
```

**Reply by danharrin** [reply]
You can always use `parent::getSaveFormAction()->label('New label')` to avoid the other repetition

**Reply by divdax** [reply]
Thank you both for your help! 🙏  I really like Filament so far. Second day of diving into it.
